### DISCUSSION TITLE
Default keybindings and Keypad cursor keys

### DISCUSSION THREAD

**Discussion by daron81** [orbitalquark/textadept#290] [General]
Hello,

The default keybindings mostly work with either the regular cursor keys, or the cursor keys on the keypad. The exceptions I've noticed are for 'Move line Up/Down', which don't work with the kp cursors.

I know it's easy enough to change, but just wondering why that is the case, and whether it can be included by default. Thanks!

**Comment by orbitalquark** [comment]
There is no reason. I don't have a keypad to test with. Feel free to submit a patch that I can review.
